workflows/clean-era5-workflow.yaml

```diff
@@ -0,0 +1,341 @@
+          - name: out-zarr
+            value: "{{ workflow.parameters.out-zarr }}"
+
+  - name: noleap
+    inputs:
+      parameters:
+      - name: in-zarr
+      - name: out-zarr
+    outputs:
+      parameters:
+      - name: out-zarr
+        value: "{{ inputs.parameters.out-zarr }}"
+    container:
+      image: downscalecmip6.azurecr.io/dodola:0.1.0
+      env:
+      - name: IN_ZARR
+        value: "{{  inputs.parameters.in-zarr }}"
+      - name: OUT_ZARR
+        value: "{{  inputs.parameters.out-zarr }}"
+      - name: PYTHONUNBUFFERED
+        value: "1"
+      # - name: AZURE_STORAGE_ACCOUNT_NAME
+      - name: AZURE_STORAGE_ACCOUNT
+        valueFrom:
+          secretKeyRef:
+            name: workerstoragecreds-secret
+            key: azurestorageaccount
+      # - name: AZURE_STORAGE_ACCOUNT_KEY
+      - name: AZURE_STORAGE_KEY
+        valueFrom:
+          secretKeyRef:
+            name: workerstoragecreds-secret
+            key: azurestoragekey
+      command: ["dodola"]
+      args:
+      - "removeleapdays"
+      - "{{ inputs.parameters.in-zarr }}"
+      - "{{ inputs.parameters.out-zarr }}"
+      resources:
+        requests:
+          memory: "42Gi"
+          cpu: "2000m"
+        limits:
+          memory: "42Gi"
+          cpu: "2000m"
+    activeDeadlineSeconds: 2700
+    retryStrategy:
+      limit: 3
+      retryPolicy: "Always"
+
+
+  - name: standardize
+    inputs:
+      parameters:
+      - name: in-zarr
+      - name: out-zarr
+    outputs:
+      parameters:
+      - name: out-zarr
+        value: "{{ inputs.parameters.out-zarr }}"
+    script:
+      image: downscalecmip6.azurecr.io/dodola:0.1.0
+      env:
+      - name: IN_ZARR
+        value: "{{  inputs.parameters.in-zarr }}"
+      - name: OUT_ZARR
+        value: "{{  inputs.parameters.out-zarr }}"
+      - name: PYTHONUNBUFFERED
+        value: "1"
+      - name: AZURE_STORAGE_ACCOUNT_NAME
+        valueFrom:
+          secretKeyRef:
+            name: workerstoragecreds-secret
+            key: azurestorageaccount
+      - name: AZURE_STORAGE_ACCOUNT_KEY
+        valueFrom:
+          secretKeyRef:
+            name: workerstoragecreds-secret
+            key: azurestoragekey
+      - name: PYTHONUNBUFFERED
+        value: "1"
+      command: [python]
+      source: |
+        import os
+        import xarray as xr
+        from adlfs import AzureBlobFileSystem
+
+        print("starting ERA-5 cleaning")
+
+        in_store_path = os.environ.get("IN_ZARR")
+        out_store_path = os.environ.get("OUT_ZARR")
+        
+        fs = AzureBlobFileSystem()
+
+        ds = xr.open_zarr(
+            fs.get_mapper(in_store_path), 
+            # chunks="auto",
+            # chunks={
+            #     "latitude": 10,
+            #     "longitude": 10,
+            #     "time": 365,
+            # }
+        )
+        print(f"opening {in_store_path}")
+
+        ds = ds.rename({"tmax": "tasmax"})
```

Comment:
Yeah, this whole `clean-era5-workflow.yaml` is what we've run with but that's since months ago. That's why itss a huge mess.

That said, I'd suggest keeping it now because this represents how the system was run. It needs to be rewritten or completely removed when the new ERA-5 data is incorporated. Depending on how you want to play this.

Reply:
Agreed. We'll have the new ERA-5 data soon hopefully.